perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ServicesManagerMethod.java

```diff
@@ -757,16 +757,24 @@ public Destination call(ApiCaller ac, Deserializer parms) throws PerunException
 		public Destination call(ApiCaller ac, Deserializer parms) throws PerunException {
 			ac.stateChangingCheck();
 
+			Destination destination;
+			
+			if(parms.contains("propagationType")) {
```

Comment:
Implementation is correct, just please double javadoc -> for each existing javadoc add new with "propagationType" param present.